### PR TITLE
added private orb content

### PR DESCRIPTION
**Pull request by thomasr8**
Added section on the difference between public and private orbs.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by tim-circleci** [comment]
LGTM. Will there be a separate sections for the CLI commands we introduced?

**Comment by thomasr8** [comment]
> LGTM. Will there be a separate sections for the CLI commands we introduced?

Hi @tim-circleci : Is there a place where @annabaker and I can get a list of these commands to add to the documentation?

**Comment by tim-circleci** [comment]
The command for adding private orbs and the command for validating private orbs are in the documentation as well:

Creating in an existing namespace,` circleci orb create --private <ns/orb>` will create a private orb

Validating: `circleci config validate --orgSlug <your-org-slug> <path/to/config.yml>`

Next week we will also add a command for listing private orbs specifically. I'll ping you when that is done.

File: jekyll/_cci2/orb-intro.md [hunk 1]
```diff
@@ -19,6 +19,32 @@ CircleCI orbs are open-source, shareable packages of parameterizable [reusable c
...
 
+## Private orbs vs. public orbs
+
+There are two different types of orbs you can use in your configuration, depending on how you want to publish your orbs. If you prefer to publish your orb internally, and not to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), you will want to use a Private Orb. However, if you want to publish your orb to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), use a Public Orb. Descriptions of each type of orb is described in the sections below.
+
+### Private orbs
+
+**Note:** Private orbs are currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+
+Orbs are reusable chunks of configuration published to the [CircleCI Orb Registry](https://circleci.com/developer/orbs). Traditionally, these orbs are authored by individuals and then published to the entire world. But what should you do if you want to author an orb, but do not want the orb published outside of your organization? Using a private orb enables you to author an orb while ensuring the following:
```
Comment (teesloane) [inline jekyll/_cci2/orb-intro.md:30]:
```suggestion
Using a private orb enables you to author an orb while ensuring the following:
```
The beginning of the document already states what is being stated here, I would remove it.

File: jekyll/_cci2/orb-intro.md [hunk 2]
```diff
@@ -19,6 +19,32 @@ CircleCI orbs are open-source, shareable packages of parameterizable [reusable c
...
+
+By choosing to use a private orb instead of a public orb, you also need to understand certain limitations inherent in using private orbs, which include:
+
+* you will be unable to use the `circleci config validate` command to validate your configuration. You may, however, paste the content of the orb into the "orbs" stanza of your configuration inline.
+
+* you cannot use private orbs from one organization in another organization's pipelines, regardless of the relationship between organizations. This means that even if you commit code and start a pipeline, and have the necessary membership in both organizations, you can use a private orb from your configuration file, but not from another orb.
+
+### Public orbs
+
+Public orbs are used by most users when authoring and publishing orbs to the [CircleCI Orb Registry](https://circleci.com/developer/orbs). When authoring a public orb, you are enabling all other users worldwide to use your orb in their own configurations.
```
Comment (teesloane) [inline jekyll/_cci2/orb-intro.md:46]:
```suggestion
Public orbs are used by most users when authoring and publishing orbs to the [CircleCI Orb Registry](https://circleci.com/developer/orbs). When authoring a public orb, you are enabling all  CircleCI users to use your orb in their own configurations.
```

**Review by teesloane — CHANGES_REQUESTED** [review]
A few wording / style suggestions as well as a syntax fix.

File: jekyll/_cci2/orb-intro.md [hunk 3]
```diff
@@ -19,6 +19,32 @@ CircleCI orbs are open-source, shareable packages of parameterizable [reusable c
 
 Published orbs can be found on our [Orb Registry](https://circleci.com/developer/orbs), or you can [author your own orb]({{site.baseurl}}/2.0/orb-author-intro/).
 
+## Private orbs vs. public orbs
+
+There are two different types of orbs you can use in your configuration, depending on how you want to publish your orbs. If you prefer to publish your orb internally, and not to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), you will want to use a Private Orb. However, if you want to publish your orb to the [CircleCI Orb Registry](https://circleci.com/developer/orbs), use a Public Orb. Descriptions of each type of orb is described in the sections below.
+
+### Private orbs
+
+**Note:** Private orbs are currently only available if you are on the https://circleci.com/pricing[Scale Plan]. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
```
[the remark this answers is not in the document]
Comment (thomasr8) [reply jekyll/_cci2/orb-intro.md]: Fixed

**Review by teesloane — APPROVED** [review]

**Pull request merged**